genesyscloud/journey_views/genesyscloud_journey_views_proxy.go

```diff
@@ -76,3 +115,44 @@ func updateJourneyViewFn(_ context.Context, p *journeyViewsProxy, viewId string,
 func deleteJourneyViewFn(_ context.Context, p *journeyViewsProxy, viewId string) (*platformclientv2.APIResponse, error) {
 	return p.journeyViewsApi.DeleteJourneyView(viewId)
 }
```

Comment:
Please delete the cache item here is the API delete was successful. Here is an example of this being done in another package - https://github.com/MyPureCloud/terraform-provider-genesyscloud/blob/main/genesyscloud/auth_division/genesyscloud_auth_division_proxy.go#L159

Reply:
Should I add in cache on create also ?

Reply:
No, there is no need

Reply:
item deleted from cache after successful deletion